src/fragments/lib/interactions/js/getting-started.mdx

```diff
@@ -63,6 +65,49 @@ Amplify.configure({
 });
 ```
 
+### Lex V2 bot
+You can create Amazon Lex V2 chatbox in Amazon Lex console. To create your bot, follow the steps shown in [Amazon Lex V2 Developer Guide](https://docs.aws.amazon.com/lexv2/latest/dg/getting-started.html).
+
+![Interactions](/images/interactions_lex_v2_console_edit_bot.png)
+
+With manual setup, you need to provide your auth credentials and bot details to configure your app:
+
```

Comment:
I would update this a little bit, you can also call `Amplify.configure` with the default export value from `aws-exports.js` and then call `Amplify.configure` with Interactions only config. I think that is more simple for customers to add this rather than change it to match this

Reply:
We can keep this as is for now - I will go over this documentation and fix it up at a later point in time.